_includes/header.html

```diff
@@ -31,7 +31,7 @@
               <a href="https://docs.bazel.build">Documentation</a>
             </li>
             <li{% if page.nav == "contribute" %} class="active"{% endif %}>
-              <a href="https://bazel.build/contributing.html">Contribute</a>
+              <a href="/contributing.html">Contribute</a>
```

Comment:
Same here, using absolute path allows us to share the same exact file between our 3 repos